File: src/main/scala-2.11/app/Main.scala
package app

import javafx.fxml.FXMLLoader
import javafx.scene.Parent

import app.action.impl.AppAction
import app.gui.controller.impl.AppController
import app.gui.view.FXML
import app.gui.view.css.CSS
import app.util.Constants

import scalafx.Includes._
import scalafx.application.JFXApp
import scalafx.application.JFXApp.PrimaryStage
import scalafx.scene.Scene

/**
  * Created by Dragos on 17.06.2016.
  */
object Main extends JFXApp {

  val resource = FXML.getClass.getResource(Constants.applicationFXML)

  stage = new PrimaryStage {
    scene = new Scene {
      stylesheets add CSS.getClass.getResource(Constants.applicationCSS).toExternalForm
      root = FXMLLoader.load(resource).asInstanceOf[Parent]
    }
  }

}

File: src/main/scala-2.11/app/Main.java
package app;

import app.gui.view.FXML;
import app.gui.view.css.CSS;
import app.util.Constants;
import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.net.URL;

/**
  * Created by Dragos on 17.06.2016.
  */
public class Main extends Application {

    public static void main(String ...args){
        launch(args);
    }

    @Override
    public void start(Stage primaryStage) throws Exception {
        URL resource = FXML.class.getResource(Constants.applicationFXML());

        Scene scene = new Scene(FXMLLoader.load(resource));
        scene.getStylesheets().add(CSS.class.getResource(Constants.applicationCSS()).toExternalForm());

        primaryStage.setScene(scene);

        primaryStage.show();
    }
}
